### PR TITLE
Fixed collection date set for both export options (connect #2363)

### PR DESCRIPTION
**Pull request by Maseline**
#### Before the PR (what is the issue or what needed to be done)
Under the reports tab, when a user selected a date for the Data Cleaning section, the same date would show up under the data analysis section.
#### The solution
Changed from DateField to DateField2 in export-reports.handlebars in order to use an abstract function for the datepicker.
Changed from one instance of exportReportAppletView in the template to multiple instances for each report section
Added reportFromDate and reportToDate properties in exportReportAppletView and set them as the valueBinding for the date input fields  in the template
Added observers for reportFromDate and reportToDate inorder to change the min  and maxDates for the datepickers 
## Checklist
* [ ] Connect the issue
* [ ] Test plan
* [ ] Copyright header
* [ ] Code formatting
* [ ] Documentation

### CODE REVIEW

File: Dashboard/app/js/lib/views/reports/export-reports-views.js [hunk 1]
```diff
@@ -190,12 +199,14 @@ FLOW.ExportReportsAppletView = FLOW.View.extend({
   }.property('FLOW.selectedControl.selectedSurveyGroup'),
 
   showDataCleaningReport: function () {
-	var opts = {}, sId = this.get('selectedSurvey');
-	FLOW.ReportLoader.load('DATA_CLEANING', sId, opts);
+  var opts = {from:this.get("reportFromDate"), to:this.get("reportToDate")};
```
Comment (muloem) [inline Dashboard/app/js/lib/views/reports/export-reports-views.js]: The code indentation in this section seems off...

File: Dashboard/app/js/lib/views/reports/export-reports-views.js [hunk 2]
```diff
@@ -190,12 +199,14 @@ FLOW.ExportReportsAppletView = FLOW.View.extend({
...
+  var sId = this.get('selectedSurvey');
+  FLOW.ReportLoader.load('DATA_CLEANING', sId, opts);
   },
 
   showDataAnalysisReport: function () {
-	var opts = {}, sId = this.get('selectedSurvey');
+    var opts = {from:this.get("reportFromDate"), to:this.get("reportToDate")};
```
Comment (muloem) [inline Dashboard/app/js/lib/views/reports/export-reports-views.js]: same here regarding the code indentation

File: Dashboard/app/js/templates/navReports/export-reports.handlebars [hunk 3]
```diff
@@ -30,50 +31,56 @@
         document.getElementById(exportName).style.display = "block";
         evt.currentTarget.className += " active";
        }
-    </script> 
+    </script>
```
Comment (muloem) [inline Dashboard/app/js/templates/navReports/export-reports.handlebars]: Please remove the whitespace...

File: Dashboard/app/js/templates/navReports/export-reports.handlebars [hunk 4]
```diff
@@ -30,50 +31,56 @@
...
       <ul class="exportSelect">
         <li class="dataCleanExp trigger" onclick="openExportOptions(event, 'dataCleanExp_options')">
           <h2>{{t _data_cleaning_export}}</h2>
           <h6>{{t _importable_back_to_akvo_flow}}</h6>
           <p class="expDescr">{{t _combines_options}}</p>
+        {{#view FLOW.ExportReportsAppletView dataCleaningSection=true}}
           <div id="dataCleanExp_options" class="options">
               <div class="dataCollectedDate">
-                <p>{{t _collection_period}}</p>
+                <p>{{t _collection_period}}</p>
```
Comment (muloem) [inline Dashboard/app/js/templates/navReports/export-reports.handlebars]: remove whitespace

Comment (Maseline) [reply Dashboard/app/js/templates/navReports/export-reports.handlebars]: @muloem most of the issues were actually not present for me locally but made a commit for those that were present

File: Dashboard/app/js/lib/views/reports/export-reports-views.js (hunk 2 above)
Comment (Maseline) [reply Dashboard/app/js/lib/views/reports/export-reports-views.js]: there was no indentation issue on my side, if you notice the line that was actually added is correctly indented, used tabs

File: Dashboard/app/js/templates/navReports/export-reports.handlebars (hunk 3 above)
Comment (Maseline) [reply Dashboard/app/js/templates/navReports/export-reports.handlebars]: there was no white space on my end, locally

File: Dashboard/app/js/templates/navReports/export-reports.handlebars (hunk 4 above)
Comment (Maseline) [reply Dashboard/app/js/templates/navReports/export-reports.handlebars]: there was no white space here either

File: Dashboard/app/js/templates/navReports/export-reports.handlebars [hunk 5]
```diff
@@ -30,50 +31,56 @@
...
        }
-    </script> 
+    </script>      
     <section class="exportContainer">
       <ul class="exportSelect">
         <li class="dataCleanExp trigger" onclick="openExportOptions(event, 'dataCleanExp_options')">
           <h2>{{t _data_cleaning_export}}</h2>
           <h6>{{t _importable_back_to_akvo_flow}}</h6>
           <p class="expDescr">{{t _combines_options}}</p>
+        {{#view FLOW.ExportReportsAppletView dataCleaningSection=true}}
```
Comment (muloem) [inline Dashboard/app/js/templates/navReports/export-reports.handlebars]: Do we still need this setting `dataCleaningSection`?

File: Dashboard/app/js/lib/views/reports/export-reports-views.js [hunk 6]
```diff
@@ -158,6 +145,20 @@ FLOW.ExportReportsAppletView = FLOW.View.extend({
...
+  reportToDate: undefined,
+  
+  setMinDate: function () {
+    if (this.get('reportFromDate') !== "") {
+      this.$(".to_date").datepicker("option", "minDate", this.get("reportFromDate"))
+    }
+  }.observes('this.reportFromDate'),
+
+  setMaxDate: function () {
+    if (this.get('reportToDate') !== "") {
```
Comment (muloem) [inline Dashboard/app/js/lib/views/reports/export-reports-views.js]: It should be enough here to say `if (this.get('reportToDate')) {...}` same as previous `reportFromDate`

**Pull request merged**